build/toolchain/host/BUILD.gn

```diff
@@ -42,3 +45,42 @@ gcc_toolchain("fake_${host_cpu}_gcc") {
     chip_fake_platform = true
   }
 }
+
+config("sysroot") {
+  cflags = [
+    "--sysroot=${sysroot}",
+    "--target=${target_triple}",
+  ]
+  ldflags = [
+    "--sysroot=${sysroot}",
+    "--target=${target_triple}",
+  ]
+}
+
+template("linux_clang_crossbuild_toolchain") {
```

Comment:
We shouldn't have "crossbuild toolchains" and regular toolchains. They are the same thing.

Reply:
(suppose I enable host + arm64 build on the Pi. What should happen? It should build once)

Reply:
The updated version will disable this as well as `enable_linux_arm[64]_clang_build` option when host platform is arm/arm64.

I'm also thinking about not disable them so the user can use a custom sysroot to make the build more reproducible.